dgp/proto/geometry.proto

```diff
@@ -77,4 +77,6 @@ message CameraIntrinsics {
   // Parameters for Double Sphere model
   double xi = 25;
 
+  // Flag to indicate camera model is Equirectangular
+  int32  equirectangular = 27;
```

Comment:
For clarification, this flag is used to indicate whether the transformation can be applied and not any particular value in the [equation](https://mathworld.wolfram.com/CylindricalEquidistantProjection.html)? If so, why `int32` instead of `bool`?

Reply:
Hello :)
Good question. The flag has similar nature as the `fisheye` flag. Hence, for consistency, I used same type. I agree a `bool` would do it.

The flag indicates whether the camera matrix shall be used with an [equirectangular projector](https://en.wikipedia.org/wiki/Equirectangular_projection) or not. i.e. the camera model